src/qt/pivx/governancewidget.cpp

```diff
@@ -142,7 +142,12 @@ void GovernanceWidget::onVoteForPropClicked(const ProposalInfo& proposalInfo)
 
 void GovernanceWidget::onCreatePropClicked()
 {
-    if (!walletModel || !governanceModel) return;
+    if (!walletModel || !governanceModel || !clientModel) return;
+
+    if (clientModel->inInitialBlockDownload()) {
```

Comment:
maybe safer to use `!governanceModel->isTierTwoSync()` to avoid the (small) possibility where IBD is false but the node is not yet at the network chain's tip, and a superblock has happened within that window?

Reply:
Works too. It minimizes the window from 24 hours to 1 hour.
Will include a proposal submission blockage for this last period as well.